src/jackdaw/data/producer.clj

```diff
@@ -12,20 +12,20 @@
 
 (defn ^ProducerRecord ->ProducerRecord
   "Given unrolled ctor-style arguments creates a Kafka `ProducerRecord`."
-  ([{:keys [:topic-name]} value]
+  ([{:keys [topic-name]} value]
```

Comment:
Doesn't `:topic-name` work?  
Are you changing it to keep it consistent?

Reply:
yes

Reply:
Take a look at https://github.com/FundingCircle/jackdaw/pull/83. We'll merge that first and you can rebase on top. Might save a few.